aten/src/ATen/native/tags.yaml

```diff
@@ -12,6 +12,12 @@
   desc: |
           This tag indicates if an operator's output's shape depends on input Tensor
           data.
+- tag: data_dependent_output
```

Comment:
To clarify, does this include operators that may error out depending on input values? (would be good to clarify in the description)

Reply:
Well, based on what the PR does, the answer is no. But... do you think it should?